File: src/com/jda/algorithm_programs/MonthlyPayment.java
package com.jda.algorithm_programs;

import com.jda.utility.Util;

public
class MonthlyPayment {

  public static
  void main(String[] args) {
    double p = Double.parseDouble(args[0]);
    double y = Double.parseDouble(args[1]);
    double r = Double.parseDouble(args[2]);

    System.out.println(Util.monthlyPayment(p, y, r));
  }
}

File: src/com/jda/algorithm_programs/MonthlyPayment.kt
package com.jda.algorithm_programs

import com.jda.utility.Util

object MonthlyPayment {

    @JvmStatic
    fun main(args: Array<String>) {
        val p = java.lang.Double.parseDouble(args[0])
        val y = java.lang.Double.parseDouble(args[1])
        val r = java.lang.Double.parseDouble(args[2])

        println(Util.monthlyPayment(p, y, r))
    }
}
